src/main/io/serial.c

```diff
@@ -492,12 +492,17 @@ void serialInit(bool softserialEnabled, serialPortIdentifier_e serialPortToDisab
             }
         }
 #endif
+        else if ((serialPortUsageList[index].identifier == SERIAL_PORT_SOFTSERIAL1
```

Comment:
Can you please make this a:

#ifdef USE_SOFTSERIAL

#else

#endif

And out the complete if statement twice... splitting the condition is terrible for readability and debugging.

Reply:
changed